pom.xml

```diff
@@ -162,47 +143,34 @@
         <version>${guava.version}</version>
       </dependency>
       <dependency>
-        <groupId>org.apache.abdera</groupId>
-        <artifactId>abdera-parser</artifactId>
```

Comment:
Yes. https://abdera.apache.org/
I am not concerned about it going away, but more interested in the technique for determining it is not needed.